modules/dr-restoring-cluster-state.adoc

```diff
@@ -652,6 +643,15 @@ AllNodesAtLatestRevision
 +
 If the output includes multiple revision numbers, such as `2 nodes are at revision 6; 1 nodes are at revision 7`, this means that the update is still in progress. Wait a few minutes and try again.
 
+. Monitor the Platform Operators by running:
```

Comment:
```suggestion
. Monitor the platform Operators by running:
```
- Does this instruction mean - RHOCP Operators? If yes then you can use `{product-title} Operators`, otherwise `platform` with lower case.
- I would have suggested to use "by running the *following command*:" but I noticed that this is not being used consistently in this file. So I guess I leave it up to you.